users/responses.py

```diff
@@ -55,9 +55,69 @@
 }
 
 verify_otp_401 = {
-  "detail": "Authentication credentials were not provided."
+  "detail": [
+      "Authentication credentials were not provided.",
+      "Invalid token."
+  ]
+}
+
+get_user_details_200 = {
+  "first_name": "Sahil",
+  "last_name": "Silare",
+  "email": "sahil@gmail.com",
+  "verified": False,
+  "contact": "",
+  "bquiz_score": 0,
+  "user_type": "GST",
+  "linkedin": "https://www.linkedin.com/in/anshsrtv",
+  "facebook": "https://www.facebook.com/ansh.srivastava.77/",
+  "applied": True,
+  "id": 1
+}
+
+resend_otp_200 = {
+    "message": "An otp has been sent to your mail id to reset your password"
```

Comment:
```suggestion
    "message": "An otp has been sent to your mail id."
```
Since the API requires you to be authorised, it doesn't make sense to use it to reset the password. 😅